imagine/main.go

```diff
@@ -319,9 +329,9 @@ func (conf *Config) CompareFields(client *pilosa.Client, dbIndex *pilosa.Index,
 		case fieldTypeBSI:
 			dbIndex.Field(name, pilosa.OptFieldTypeInt(int64(field.Min), int64(field.Max)))
 		case fieldTypeSet:
-			dbIndex.Field(name, pilosa.OptFieldTypeSet("none", 0))
+			dbIndex.Field(name, pilosa.OptFieldTypeSet(pilosa.CacheType(field.Cache.String()), 1000))
```

Comment:
need to add option for setting cache size with sane default. Also need to add option for ranked cache, not just LRU